BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/SkontoViewModel.swift

```diff
@@ -138,4 +148,9 @@ class SkontoViewModel {
         let skontoPercentage = ((priceWithoutSkonto.value - priceWithSkonto.value) / priceWithoutSkonto.value) * 100
         skontoValue = Double(truncating: skontoPercentage as NSNumber)
     }
+
+    private func calculateSavedAmount() -> Price {
```

Comment:
[suggestion] Here as well. Maybe we can rename it like: "calculateSkontoSavingsAmount".